### PR TITLE
Create bug_new.yml

### PR DESCRIPTION
**Pull request by tejavar**
Trying a new beta feature - Issue Template for new bug creation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_new.yml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  - type: input
+    id: version-terraform
+    attributes:
+      label: Terraform
+      description: Please provide the Terraform version.
+      placeholder: e.g. v1.1.5
+    validations:
+      required: true
+  - type: input
+    id: version-terraform-provider-vsphere
```
Comment (megan07) [inline .github/ISSUE_TEMPLATE/bug_new.yml:35]:
```suggestion
    id: version-terraform-provider-google
```

**Review by megan07 — APPROVED** [review]
I think this looks good - just want to check with @rileykarson if he has any feedback on switching up the new. bug template as well.

File: .github/ISSUE_TEMPLATE/bug_new.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
+name: "Bug Report"
+description: Is something not working as expected? Submit a bug.
+title: "REPLACE WITH A SHORT DESCRIPTION"
```
Comment (rileykarson) [inline .github/ISSUE_TEMPLATE/bug_new.yml:3]: Should this have a description entered?

**Review by rileykarson — COMMENTED** [review]
Neat! No concerns on my end, although we'll want to make sure to clean up the old template.

**Comment by rileykarson** [comment]
We have this nowadays: https://github.com/hashicorp/terraform-provider-google/issues/new?assignees=&labels=bug&projects=&template=00_bug.yml

**Pull request closed without merging**